lms/templates/admin/course/show.html.jinja2

```diff
@@ -4,26 +4,26 @@
 {% block content %}
 <div>
     <fieldset>
-        <legend class="label has-text-centered">Course</legend>
-            {{ macros.disabled_text_field("ID", course.id) }}
-            {{ macros.disabled_text_field("Name", course.name) }}
-            {{ macros.disabled_text_field("H ID", course.authority_provided_id) }}
-            {{ macros.created_updated_fields(course) }}
-    </fieldset>
-
-    <fieldset>
-        <legend class="label has-text-centered">Assignments</legend>
-        {{ macros.assignments_table(request, course.assignments) }}
+          {{ macros.disabled_text_field("ID", course.id) }}
+          {{ macros.disabled_text_field("Name", course.name) }}
+          {{ macros.disabled_text_field("H ID", course.authority_provided_id) }}
+          {{ macros.disabled_text_field("Context ID", course.lms_id) }}
+          {{ macros.created_updated_fields(course) }}
     </fieldset>
 
     <fieldset class="box mt-6">
       <legend class="label has-text-centered">Application Instance</legend>
       {{ macros.instance_preview(request, course.application_instance) }}
     </fieldset>
+
     <fieldset class="box mt-6">
         <legend class="label has-text-centered">Organization</legend>
         {{ macros.organization_preview(request, course.application_instance.organization) }}
     </fieldset>
 
+    <fieldset class="box mt-6">
```

Comment:
This was missing the class.

Also moved to the bottom. We could add the section navigation once we start adding more things here.